packages/components-js/tests/vrt/puppeteer/specs/table.vrt.ts

```diff
@@ -43,7 +35,18 @@ xit('should have no visual regression for :hover + :focus-visible', async () =>
     </p-table-row>
   </p-table-body>
 </p-table>`;
-      await setContentWithDesignSystem(page, getBodyMarkup(getElementsMarkup));
+
+it.each(defaultViewports)('should have no visual regression for viewport %s', async (viewport) => {
```

Comment:
Ca we do a `it.each<Theme>(['light', 'dark'])(...)` instead of duplicating the entire test?

Reply:
@denyo When it was `describe.each<Theme>(['light', 'dark'])(...)`, vrt tests were failing all the time (every time different state).. So we discussed it in daily and I split it into 2 tests (because it started failing when I added dark theme)

Reply:
Got it, fixed it